src/amo/components/UserReview/styles.scss

```diff
@@ -23,6 +23,12 @@
   .ShowMoreCard.UserReview-emptyBody {
     margin: 0;
   }
+
+  @include respond-to(medium) {
+    .TooltipMenu {
+      width: fit-content;
```

Comment:
move this within the `.TooltipMenu` block above so don't need to repeat the selector here?

Reply:
while doing this, I noticed we don't even the default `width` above.